src/index/blockfilterindex.cpp

```diff
@@ -31,6 +31,8 @@ constexpr char DB_FILTER_POS = 'P';
 constexpr unsigned int MAX_FLTR_FILE_SIZE = 0x1000000; // 16 MiB
 /** The pre-allocation chunk size for fltr?????.dat files */
 constexpr unsigned int FLTR_FILE_CHUNK_SIZE = 0x100000; // 1 MiB
+/** Maximum size of the cfheaders cache */
```

Comment:
Nit: give justification in comments for magic numbers so future readers have some idea how this was chosen and if/when it will need to be adjusted.

Reply:
done